src/App.js

```diff
@@ -1,19 +1,30 @@
 import React from 'react';
-import { Routes, Route } from 'react-router-dom';
+import { Route, Switch } from 'react-router-dom';
+import './pages/App.css';
+import Calculator from './components/calculator';
+import Navbar from './components/Navbar';
 import Home from './pages/Home';
 import Quote from './pages/Quote';
-import Calculator from './components/calculator';
-import Navbar from './pages/Navbar';
+import NotMatch from './pages/NotMatch';
 
 function App() {
   return (
     <>
       <Navbar />
-      <Routes>
-        <Route path="/quote" element={<Quote />} />
-        <Route path="/home" element={<Home />} />
-        <Route path="/calculator" element={<Calculator />} />
-      </Routes>
+      <Switch>
+        <Route exact path="/">
+          <Home />
+        </Route>
+        <Route path="/calculator">
+          <Calculator />
+        </Route>
+        <Route path="/quote">
+          <Quote />
+        </Route>
+        <Route path="*">
+          <NotMatch />
+        </Route>
+      </Switch>
```

Comment:
- [ ] I think here you are using react-router-dom but you forget to install it npm insstall react-router-dom so that make me unable to see your website I highly recommend adding it to your project to let it work properly 🚀💯🔥


![image](https://user-images.githubusercontent.com/58553711/201932858-393d9b9c-ae05-4a0e-a9c0-983a31d47967.png)